iina/AppDelegate.swift

```diff
@@ -238,9 +238,13 @@ class AppDelegate: NSObject, NSApplicationDelegate {
         }
       }
 
-      // enter PIP
-      if #available(macOS 10.12, *), let pc = lastPlayerCore, commandLineStatus.enterPIP {
-        pc.mainWindow.enterPIP()
+      if let pc = lastPlayerCore {
+        // PiP is not supported in music mode. Ignore the PiP option if music mode was specified.
```

Comment:
Can you add this to the help message?

Reply:
Happy to update the help. Something like this?
```text
    --music-mode:
            Enter music mode after opening the media.
            If --music-mode is specified the --pip option will be ignored.
```

What I thought was really needed was option validation where if you use both then IINA fails with an error. The typical style error message would be:
```text
Cannot specify both --music-mode and --pip
```

I did not add that because validation is currently a missing feature of the IINA CLI. For example there is no attempt to reject mpv options that IINA does not support like the ones that configure mpv's basic controller. IINA ignores the options.

Should I just update the help, or also add validation and treat this as a fatal error like most command line utilities would do?

Reply:
+1 to failing with a useful error message. Otherwise it's not obvious why --music-mode should be favored over --pip